api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -279,7 +300,22 @@ message CertificateValidationContext {
   // directory for any file moves to support rotation. This currently only
   // applies to dynamic secrets, when the *CertificateValidationContext* is
   // delivered via SDS.
-  config.core.v3.DataSource trusted_ca = 1;
+  //
+  // Only one of *trusted_ca*, *ca_certificate_provider*, and *ca_certificate_provider_instance*
```

Comment:
I don't see `ca_certificate_provider` defined in this struct. Either that needs to be defined here or the comment should be updated to reflect it. IIRC @coolnsu wanted to keep it for future use.